doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -373,8 +373,8 @@ set with the new rcParameter :rc:`axes.titley`.
 .. plot::
 
     fig, axs = plt.subplots(1, 2, constrained_layout=True, figsize=(5, 2))
-    axs[0].set_title('y=0.7\n$\sum_{j_n} x_j$', y=0.7)
-    axs[1].set_title('y=None\n$\sum_{j_n} x_j$')
+    axs[0].set_title(r'y=0.7\n$\sum_{j_n} x_j$', y=0.7)
+    axs[1].set_title(r'y=None\n$\sum_{j_n} x_j$')
```

Comment:
This will print with `\n` in the image. https://output.circle-artifacts.com/output/job/1fdad817-89a0-4227-9235-0f0e242fec00/artifacts/0/doc/build/html/users/prev_whats_new/whats_new_3.3.0.html#axes-set-title-gains-a-y-keyword-argument-to-control-auto-positioning

I take it that this is the py312 issue, so not sure if this is the solution, but 
```suggestion
    axs[0].set_title('y=0.7\n$\\sum_{j_n} x_j$', y=0.7)
    axs[1].set_title('y=None\n$\\sum_{j_n} x_j$')
```
?